dbms/src/Interpreters/AsynchronousMetrics.cpp

```diff
@@ -108,8 +108,8 @@ void AsynchronousMetrics::update()
     {
         if (auto mark_cache = context.getMarkCache())
         {
-            set("MarkCacheBytes", mark_cache->weight());
-            set("MarkCacheFiles", mark_cache->count());
+            set("MarkCacheBytes", mark_cache->getSizeInUse());
+            set("MarkCacheFiles", mark_cache->getUsedRegionsCount());
```

Comment:
We have the pages that are currently locked for use and cannot be evicted, and the pages that are not used -- cached pages, actually. So we can show these two numbers -- MarkCacheLockedBytes (Active/UsedBytes?) and MarkCacheCachedBytes (Reserved/Total/whateverBytes). Not sure about the names. And for files, are they still files or maybe something more abstract, like entries?

Reply:
1) Ok, in the next commit I will change these names to `MarkCacheUsedBytes` (total used size NOT accounting the metadata) and `MarkCacheCachedEntries` (because there are no real files).